### PR TITLE
@alloy - More artsy primer QA

### PR DESCRIPTION
**Pull request by broskoski**
Incidentally closes https://github.com/artsy/collector-experience/issues/193

### CODE REVIEW

File: desktop/apps/artsy_primer/personalize/stylesheets/categories.styl [hunk 1]
```diff
@@ -75,11 +80,17 @@ category-size = 400px
...
+      width 100%
+      height 100vw
   .artsy-primer-personalize-following-icon
     position absolute
     width 400px
     height 400px
     padding-top 45%
+    @media screen and (max-width: 768px)
+      width 100%
+      height 100%
```
Comment (alloy) [inline desktop/apps/artsy_primer/personalize/stylesheets/categories.styl]: Just double-checking, is it correct that you’re not using the viewport height here?

**Comment by alloy** [comment]
Looks primo 👌

File: desktop/apps/artsy_primer/personalize/stylesheets/categories.styl (hunk 1 above)
Comment (broskoski) [inline desktop/apps/artsy_primer/personalize/stylesheets/categories.styl]: Indeed, that is correct. It just needs to fill the container.

Comment (alloy) [inline desktop/apps/artsy_primer/personalize/stylesheets/categories.styl]: 👍

**Pull request merged**